specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-11-01/extensions.json

```diff
@@ -0,0 +1,713 @@
+              "description": "Currently installed version of the extension.",
+              "type": "string",
+              "readOnly": true,
+              "x-nullable": true
+            },
+            "provisioningState": {
+              "description": "Status of installation of this extension.",
+              "type": "string",
+              "readOnly": true,
+              "$ref": "../../common/2022-03-01/definitions.json#/definitions/ProvisioningStateDefinition"
+            },
+            "statuses": {
+              "description": "Status from this extension.",
+              "type": "array",
+              "x-nullable": true,
+              "items": {
+                "$ref": "#/definitions/ExtensionStatus"
+              },
+              "x-ms-identifiers": []
+            },
+            "errorInfo": {
+              "description": "Error information from the Agent - e.g. errors during installation.",
+              "type": "object",
+              "readOnly": true,
+              "x-nullable": true,
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorDetail"
+            },
+            "customLocationSettings": {
+              "description": "Custom Location settings properties.",
+              "type": "object",
+              "readOnly": true,
+              "x-nullable": true,
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "packageUri": {
+              "description": "Uri of the Helm package",
+              "type": "string",
+              "readOnly": true,
+              "x-nullable": true
+            },
+            "aksAssignedIdentity": {
+              "description": "Identity of the Extension resource in an AKS cluster",
+              "x-nullable": true,
+              "type": "object",
+              "properties": {
+                "principalId": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The principal ID of resource identity."
+                },
+                "tenantId": {
+                  "readOnly": true,
+                  "type": "string",
+                  "description": "The tenant ID of resource."
+                },
+                "type": {
+                  "type": "string",
+                  "description": "The identity type.",
+                  "enum": [
+                    "SystemAssigned",
+                    "UserAssigned"
+                  ],
+                  "x-ms-enum": {
+                    "name": "AKSIdentityType",
+                    "modelAsString": false
+                  }
+                }
+              }
+            },
+            "isSystemExtension": {
+              "readOnly": true,
+              "description": "Flag to note if this extension is a system extension",
+              "type": "boolean",
+              "default": false
+            }
```

Comment:
Consider if you can use enum over boolean

Reply:
The property is a read-only property and we do not foresee new values.